src/index.js

```diff
@@ -0,0 +1,8 @@
+import './index.css';
+import addUser from './modules/AddUser.js';
+
+const submitBtn = document.getElementById('submitBtn');
+
+submitBtn.addEventListener('click', () => {
+  addUser();
+});
```

Comment:
| @pushkar1995 Form | Wireframe Form |
|-|-|
| ![image](https://github.com/pushkar1995/leaderboard/assets/93039359/d1d38b91-af12-4e60-a68d-39d8fde85d93) | ![image](https://github.com/pushkar1995/leaderboard/assets/93039359/65130a7c-69a4-47e8-bb50-ae18c95c1a44)) |

- [ ] It would be great if you follow the given wireframe because right now your button and heading are not don't the edges 😨 Kindly put them at the both ends to follow the given wireframe to make your project more professional 👍